File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        unordered_map<int, int> res;

        for(int num : nums){
            res[num]++;
        }
        
        vector<pair<int,int>> arr;

        for(const auto& pair : res){
            arr.push_back({pair.second, pair.first});
        }
        sort(arr.rbegin(), arr.rend());

        vector<int> result;
        for(int i=0; i<k ;i++){
            result.push_back(arr[i].second);
        }
        

        return result;
    }
};

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        count = 0
        sol = 0
        for num in nums:
            if num == 0:
                sol = max(count, sol)
                count = 0
            else:
                count += 1
        
        return max(count, sol)
            